testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/common/hadoop/apply-config-overrides.sh

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env bash
```

Comment:
Why not to drop it all together?

Reply:
I guess to be sure that we do not have overrides in one of the base images

Reply:
To be sure no-one tries to use mechanism that we're decomissioning.
Otherwise it could be possible that test run with a different configuration than expected.

Reply:
Removal should be a follow-up sometime later.